esp-wifi/src/common_adapter/common_adapter_esp32.rs

```diff
@@ -81,7 +81,7 @@ pub(crate) unsafe fn phy_enable() {
 
 #[allow(unused)]
 pub(crate) unsafe fn phy_disable() {
-    let count = PHY_ACCESS_REF.fetch_sub(1, Ordering::SeqCst);
+    let count = (*core::ptr::addr_of_mut!(PHY_ACCESS_REF)).fetch_sub(1, Ordering::SeqCst);
```

Comment:
`PHY_ACCESS_REF` is an AtomicU32, it shouldn't need to be mutable in the first place. We shouldn't need to access it through pointer indirection.